### DISCUSSION TITLE
Next steps docs

### DISCUSSION THREAD

**Discussion by kenhorn** [superwerker/superwerker#184] [Q&A]
Having deployed SW from the quickstart I now realise I'm missing some context on how to use CT / AWS SSO based accounts. I have not used either before, and on trying to use the Account Factory and set up a workload account, I find myself a bit disorientated. I've tried reading some of the AWS docs, but am not finding them intuitive / relatable to my current AWS knowledge -- the complexity of permissions delegation in particular.

Some questions I'd like to work out answers to:
* I have a Root account, and a Root user. I created an IAM Admin user which executed SW quickstart. Should this user be used for general maintenance / deployment, or only for 'shared / CT / SSO admin' and use SSO based users for workload account creation etc?
*  Should I create top level OUs in CT for workload accounts or use Custom as the parent?
* Is there a guide to setting up a workload account, creating an AWSPowerUserAccess within it? 
* Is SSO needed for this or can I / should I use an account from the root account, which switches role into a workload role?

I suppose the ask is how does the day-to-day AWS admin - workload deployments, granting developer access, monitoring change when using the quickstart? (Are the terms SW and quickstart interchangeable here?)

Apologies if I'm repeating questions already answered / found elsewhere, am a bit mixed up between twitter, issues, PRs, discussions etc

**Comment by s0enke** [comment]
Thanks for the feedback, much appreciated! Very good questions! Will look into this within the next days!
